ideascube/serveradmin/tests/test_catalog.py

```diff
@@ -319,6 +326,32 @@ def test_remove_staticsite(staticsite_path, install_dir):
     assert root.check(exists=False)
 
 
+def test_install_zippedmedia(zippedmedia_path, install_dir, storage_dir):
```

Comment:
You don't use `storage_dir` here, but I assume it's required to setup the settings etc.

Isn't there a way to say this test function requires this fixture, without passing it as a parameter?

As it is, I'm worried that in a few months, someone sees the parameter is unused, and decides to remove it. :worried: